<!DOCTYPE html>
<html lang="en-US">

<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="favicon2.ico" />

  <link href="https://fonts.googleapis.com/css?family=Montserrat:300" rel="stylesheet">
  <link rel="stylesheet" href="stylesheets/styles.css">
  <link rel='stylesheet' href='assets/css/custom.css'>
</head>

<body>
  <style>
    body {
      background-color: #e3e6ff
    }
  </style>
  <div class="wrapper">
    <header>
      <!---->
      <center><img src="new_headshot.jpeg" alt="Ben" width='400px' /></center>
      <!---->
      <h1 style="text-align:center;"><a href="https://bharris12.github.io/">
          <font color="black">Benjamin D. Harris</font>
        </a></h1>

      <!-- <ul class="downloads">
            <li><a href="https://github.com/bharris12"><strong>Github</strong></a></li>
          </ul> -->


      <p style="text-align:center;">Contact me at:<br> bdharris17 at gmail dot com<br>or on <br><a
          href="https://twitter.com/b_harris_12">twitter</a> <br><a href="Harris_Benjamin_CV_cb_6_11.pdf">CV</a></p>

    </header>
    <section align="justify">

      <h1>About Me</h1>
      <p style="max-width:1000px;">I am a computational biologist interested in genetics, transcriptomics, and
        development.
        I have led projects studying the functional landscape of neurons (Harris et al 2021) and the
        developmental trajectory of hematopoiesis (in preparation). I made significant contributions
        to a collaboration studying maize meristem development (Xu et al 2021) and published software
        central to all of the above projects (Fisher et al 2021). I specialize in applying traditional
        statistical and machine learning methods in a robust meta-analytic framework. I use parallel,
        distributed, and GPU computing to process and analyze large scale genomics data. </p>


      <p style="max-width:1000px;"> I started my research career at the National Institutes of Health in the lab of Dr.
        William
        F. Simonds, studying neuroendocrine signal transduction using molecular biology and behavior.
        I then transitioned into computational biology with an internship at the Icahn School of Medicine
        in the lab of Dr. Eric Sobie. There I modeled cardiac action potentials using CUDA GPU computing in
        order to understand the paramters that are sensitive to an arythmia. I then went ot Cold Spring Harbor
        laboratory for a summer internship with Dr. Mickey Atwal. I searched for novel antigens expressed exclusively

        in cancer and immune priveleged tissues that could be used for new cancer therapeutics.
      </p>

      <h1>Publications</h1>
      <p><i>Single-cell co-expression analysis reveals that transcriptional modules are shared across cell types in the
          brain.</i> <br>
        <b>Benjamin D. Harris</b>, Megan Crow, Stephan Fischer, and Jesse Gillis. <br>
        Cell Systems, 2021 <a href="https://doi.org/10.1016/j.cels.2021.04.010">doi: 10.1016/j.cels.2021.04.010</a>
      </p>

      <p>
        <i>Scaling up reproducible research for single cell transcriptomics using MetaNeighbor.</i><br>
        Stephan Fischer, Megan Crow, <b>Benjamin D. Harris</b>, and Jesse Gillis. <br>
        Nature Protocols, 2021 <a href="https://doi.org/10.1038/s41596-021-00575-5"> doi: 10.1038/s41596-021-00575-5</a>
      </p>

      <p>
        <i>Single-cell RNA sequencing of developing maize ears facilitates functional analysis and trait candidate gene
          discovery.</i><br>
        Xiaosa Xu, Megan Crow, Brian R. Rice, Forrest Li, <b>Benjamin D. Harris </b>..., Jesse Gillis, and David
        Jackson. <br>
        Developmental Cell, 2021 <a href="https://doi.org/10.1016/j.devcel.2020.12.015">doi:
          10.1016/j.devcel.2020.12.015 </a>
      </p>

      <p>
        <i>Ten simple rules for getting the most out of a summer laboratory internship</i><br>
        Toby P. Aicher, Dániel L. Barabási, <b>Benjamin D. Harris</b>, Ajay Nadig, Kaitlin L. Williams<br>
        Plos Computational Biology 2017, <a href='https://doi.org/10.1371/journal.pcbi.1005606'>doi:
          10.1371/journal.pcbi.1005606</a>
      </p>
      <p>
        <i>A central role for R7bp in the regulation of itch sensation</i><br>
        Mritunjay Pandey, Jian-Hua Zhang, Santosh K. Mishra, Poorni R. Adikaram, <b>Benjamin Harris</b>...William F.
        Simonds<br>
        Pain, 2017 <a
          href="https://journals.lww.com/pain/Abstract/2017/05000/A_central_role_for_R7bp_in_the_regulation_of_itch.20.aspx">doi:doi:
          10.1097/j.pain.0000000000000860</a>

      </p>
      <p>
        <i>Improving the specificity and efficacy of CRISPR/CAS9 and gRNA through target specific DNA reporter</i><br>
        Jian-Hua Zhang, Mritunjay Pandey, John F. Kahler, Anna Loshakov, <b>Benjamin D. Harris</b>, ..., William F.
        Simonds<br>
        Journal of Biotechnology, 2014 <a href="https://doi.org/10.1016/j.jbiotec.2014.08.033">doi:
          10.1016/j.jbiotec.2014.08.033 </a>
      </p>
      <h1>Software</h1>
      <p>Python MetaNeighbor : a method to rapidly assess cell type identity using both functional and random gene sets
        <br>
        Github : <a href="https://github.com/gillislab/pyMn">pyMN</a>
      </p>
      <h1>Teaching</h1>
      <h4>Summer 2021 CSHL Intern Python Programming Course</h4>
      <p>
        I contributed 3 lectures to the 10 lecture course, linked below <br>
        <span class="tab"></span><b>Lecture 3: </b><a
          href="https://github.com/bharris12/URP_2021_Programming_Course/tree/main/lecture_3">Pandas</a><br>
        <span class="tab"></span><b>Lecture 5: </b><a
          href="https://github.com/bharris12/URP_2021_Programming_Course/tree/main/lecture_5">Pandas 2 +
          Statistics</a><br>
        <span class="tab"></span><b>Lecture 6: </b><a
          href="https://github.com/bharris12/URP_2021_Programming_Course/tree/main/lecture_6">Genomics</a>
      </p>

      <p>
        I co-organize a seminar series in the CSHL quantiative biology department called "Practical Programming" <br>
        <span class="tab">The seminars have been on topics including <br>
          <span class="tab">Multiprocessing multithreading and vectorization<br>
            Style and Organization in Python<br>
            Elzar Batch System Usage <br>
            Snakemake<br>
            Python packages<br>
            Optimizing for performance/speed (matlab, python)<br>
            Github workflow<br>
            Unit Testing<br>
            Personal Web dev<br></span></span>
        <br>
        <span class="tab">I taught the Multiprocessing multithreading and vectorization lecture. <br>
          <span class="tab"></span>The materials are in this <a
            href="https://github.com/bharris12/practical_progamming_lecture_1">repository</a></span>
      </p>

      <h1>Misc</h1>
      <p>In my free time you can find me in nature...</p>



      <table width="100%" border="0">
        <tr>
          <td><img src="image1.JPG" alt="" align="left" width='400px' /></td>
          <td><img src="image2.JPG" alt="" align="center" width='400px' /></td>
          <td><img src="image3.JPG" alt="" align="right" width='400px' /></td>
        </tr>
      </table>
    </section>

    </section>
    </section>
  </div>
  <script src="/assets/js/scale.fix.js"></script>
</body>

</html>