<!DOCTYPE html>
<html lang="en-US">

<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" type="image/x-icon" href="new_headshot.jpeg">

  <link href="https://fonts.googleapis.com/css?family=Montserrat:300" rel="stylesheet">
  <link rel="stylesheet" href="stylesheets/styles.css">
  <link rel='stylesheet' href='assets/css/custom.css'>
</head>

<body>
  <style>
    body {
      background-color: #e3e6ff
    }
  </style>
  <div class="wrapper">
    <header class="site-header-content">
      <div class="profile-container">
        <img src="new_headshot.jpeg" alt="Ben" class="profile-img" />
      </div>
      <h1 class="site-title-text"><a href="https://bharris12.github.io/">
          Benjamin D. Harris
        </a></h1>

      <p class="contact-text">Contact me at:<br> bdharris17 at gmail dot com<br>or on <br><a
          href="https://twitter.com/b_harris_12">twitter</a> <br><a href="Harris_Benjamin_CV_cb.pdf">CV</a></p>

    </header>
    <section class="main-content">

      <h1>About Me</h1>
      <p style="max-width:1000px;">I am a computational biologist interested in genetics, transcriptomics, and
        development.
        I have led projects studying the functional landscape of neurons (Harris et al 2021) and the
        developmental trajectory of hematopoiesis (in preparation). I made significant contributions
        to a collaboration studying maize meristem development (Xu et al 2021) and published software
        central to all of the above projects (Fisher et al 2021). I specialize in applying traditional
        statistical and machine learning methods in a robust meta-analytic framework. I use parallel,
        distributed, and GPU computing to process and analyze large scale genomics data. </p>


      <p style="max-width:1000px;"> I started my research career at the National Institutes of Health in the lab of Dr.
        William
        F. Simonds, studying neuroendocrine signal transduction using molecular biology and behavior.
        I then transitioned into computational biology with an internship at the Icahn School of Medicine
        in the lab of Dr. Eric Sobie. There I modeled cardiac action potentials using CUDA GPU computing in
        order to understand the paramters that are sensitive to an arythmia. I then went ot Cold Spring Harbor
        laboratory for a summer internship with Dr. Mickey Atwal. I searched for novel antigens expressed exclusively

        in cancer and immune priveleged tissues that could be used for new cancer therapeutics.
      </p>

      <h1>Work Experience</h1>
      <p>
        <b>Stealth AI x Immunology Company</b><br>
        <i>Consultant Oct 2024 - Present</i><br>
        Conduct strategy, scientific, and competitive intelligence research to advance a pre-seed stealth immunology company
      </p>
      <p>
        <b>Lyell Immunopharma</b><br>
        <i>Senior Computational Biologist November 2021 - Present</i><br>
        <ul>
          <li>Integrated clinical, translational, and sequencing data to support development of LYL314 (CD19/CD20 CAR-T) into 2 pivotal trials</li>
          <li>Collaborated with research, process sciences, and MSAT teams on design, execution, analysis, and presentation of experiments that impact the direction of Tumor Infiltrating Lymphocytes (TIL) program</li>
          <li>Built pipelines for bulkTCRseq and scRNA/TCR/CITEseq to standardize data analysis across the computational team</li>
          <li>Co-authored regulatory documents (IND, RMAT, EOP1) and 7 clinical/research conference presentations across 3 clinical programs (presenter for 1)</li>
        </ul>
      </p>
      <h1>Publications</h1>
      <div class="bibliography-scroll">
      <!-- SCHOLAR_START -->

      <p><i>Large-scale single-cell profiling of stem cells identifies redundant regulators of shoot development and yield trait variation</i> <br>
        Xiaosa Xu and Michael Passalacqua and Brian Rice and Edgar Demesa-Arevalo and Mikiko Kojima and Yumiko Takebayashi and Xingyao Yu and <b>Benjamin Harris</b> and Yuchen Liu and Andrea Gallavotti and Hitoshi Sakakibara and Jesse Gillis and David Jackson <br>
        Developmental Cell, 2025 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:UebtZRa9Y70C">[Google Scholar]</a>
      </p>

      <p><i>Optimizing Stim-RTM, a synthetic stimulatory agent, for feeder-free tumor-infiltrating lymphocyte manufacturing</i> <br>
        Riley P Allen and Jason Duecker and Mikayla Min and Samantha Chan and Christina Ta and <b>Benjamin Harris</b> and Emily Fu-Sum and Stefan Siebert and Purnima Sundar and Yogin Patel and Shobha Potluri and Suman Vodnala and Gary Lee and Alexander Cheung and Aileen Li <br>
        Cancer Research, 2025 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:Se3iqnhoufwC">[Google Scholar]</a>
      </p>

      <p><i>Large-scale single-cell profiling of stem cells uncovers redundant regulators of shoot development and yield trait variation</i> <br>
        Xiaosa Xu and Michael Passalacqua and Brian Rice and Edgar Demesa-Arevalo and Mikiko Kojima and Yumiko Takebayashi and <b>Benjamin Harris</b> and Hitoshi Sakakibara and Andrea Gallavotti and Jesse Gillis and David Jackson <br>
        BioRxiv, 2024 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:WF5omc3nYNoC">[Google Scholar]</a>
      </p>

      <p><i>440 Utilizing Stim-RTM technology to reduce live feeder cells in the tumor infiltrating lymphocyte culture process</i> <br>
        Riley P Allen and Jason Duecker and Mikayla Min and Samantha Chan and Christina Ta and Emily Fu-Sum and <b>Benjamin Harris</b> and Purnima Sundar and Yogin Patel and Aileen Li and Shobha Potluri and Suman Vodnala and Gary Lee and Alexander Cheung <br>
        Journal for ImmunoTherapy of Cancer 12 (Suppl 2), 2024, 2024 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:ufrVoPGSRksC">[Google Scholar]</a>
      </p>

      <p><i>Identification of pan-cancer/testis genes and validation of therapeutic targeting in triple-negative breast cancer: Lin28a-based and Siglece-based vaccination induces antitumor immunity and inhibits metastasis</i> <br>
        Jason A Carter and Bharati Matta and Jenna Battaglia and Carter Somerville and <b>Benjamin D. Harris</b> and Margaret LaPan and Gurinder S Atwal and Betsy J Barnes <br>
        Journal for immunotherapy of cancer, 2023 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:zYLM7Y9cAGgC">[Google Scholar]</a>
      </p>

      <p><i>379 Epi-RTM P2 protocol produces a scalable polyclonal TIL product with a greater expansion success rate across hot and cold tumors in shorter culture time</i> <br>
        Yogin Patel and Jaewon Yun and Melissa Bedard and Melissa DeFrancesco and Carson Harms and <b>Benjamin Harris</b> and Saeed Azimi and Ngoc-Han Ha and Audrey Garces and Shobha Potluri and Gary Lee and Suman Vodnala <br>
        Journal for ImmunoTherapy of Cancer 11 (Suppl 1), 2023, 2023 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:YsMSGLbcyi4C">[Google Scholar]</a>
      </p>

      <p><i>370 The Epi-RTM technology produces a polyclonal TIL product (LYL845) with a greater expansion success rate across hot and cold tumors, improved product phenotype, and maintenance of TCR diversity</i> <br>
        Yogin Patel and <b>Benjamin Harris</b> and Melissa Bedard and Joanna Kritikou and Meritxell Galindo Casas and Carson Harms and Melissa DeFrancesco and Purnima Sundar and Nicholas Restifo and Gary Lee and Shobha Potluri and Suman Kumar Vodnala <br>
        Journal for ImmunoTherapy of Cancer 10 (Suppl 2), 2022, 2022 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:Y0pCki6q_DkC">[Google Scholar]</a>
      </p>

      <p><i>340 The Epi-RTM technology produces a polyclonal TIL product (LYL845) with diverse tumor-reactive clones that have stem-like qualities and anti-tumor function</i> <br>
        <b>Benjamin Harris</b> and Yogin Patel and Ngoc-Han Ha and Lora Zhao and Sheila Lou and Stefan Siebert and Emily Fu-Sum and Rigel Kishton and Purnima Sundar and Suman Kumar Vodnala and Shobha Potluri <br>
        Journal for ImmunoTherapy of Cancer 10 (Suppl 2), 2022, 2022 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:W7OEmFMy1HYC">[Google Scholar]</a>
      </p>

      <p><i>Single-cell RNA sequencing of developing maize ears facilitates functional analysis and trait candidate gene discovery</i> <br>
        Xiaosa Xu and Megan Crow and Brian R Rice and Forrest Li and <b>Benjamin Harris</b> and Lei Liu and Edgar Demesa-Arevalo and Zefu Lu and Liya Wang and Nathan Fox and Xiaofei Wang and Jorg Drenkow and Anding Luo and Si Nian Char and Bing Yang and Anne W Sylvester and Thomas R Gingeras and Robert J Schmitz and Doreen Ware and Alexander E Lipka and Jesse Gillis and David Jackson <br>
        Developmental Cell, 2021 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:9yKSN-GCB0IC">[Google Scholar]</a>
      </p>

      <p><i>Single-cell co-expression analysis reveals that transcriptional modules are shared across cell types in the brain</i> <br>
        <b>Benjamin D. Harris</b> and Megan Crow and Stephan Fischer and Jesse Gillis <br>
        Cell systems, 2021 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:u5HHmVD_uO8C">[Google Scholar]</a>
      </p>

      <p><i>Scaling up reproducible research for single-cell transcriptomics using MetaNeighbor</i> <br>
        Stephan Fischer and Megan Crow and <b>Benjamin D. Harris</b> and Jesse Gillis <br>
        Nature Protocols 16 (8), 4031-4067, 2021, 2021 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:u-x6o8ySG0sC">[Google Scholar]</a>
      </p>

      <p><i>A meta-analytic single-cell atlas of mouse bone marrow hematopoietic development</i> <br>
        <b>Benjamin D. Harris</b> and John Lee and Jesse Gillis <br>
        BioRxiv, 2021 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:UeHWp8X0CEIC">[Google Scholar]</a>
      </p>

      <p><i>Atlas level scRNAseq analysis reveals the functional landscape of cell types</i> <br>
        <b>Benjamin D. Harris</b> <br>
        Cold Spring Harbor Laboratory, 2021, 2021 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:IjCSPb-OGe4C">[Google Scholar]</a>
      </p>

      <p><i>A central role for R7bp in the regulation of itch sensation</i> <br>
        Mritunjay Pandey and Jian-Hua Zhang and Santosh K Mishra and Poorni R Adikaram and <b>Benjamin Harris</b> and John F Kahler and Anna Loshakov and Roxanne Sholevar and Allison Genis and Claire Kittock and Juraj Kabat and Sundar Ganesan and Richard R Neubig and Mark A Hoon and William F Simonds <br>
        Pain, 2017 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:2osOgNQ5qMEC">[Google Scholar]</a>
      </p>

      <p><i>Ten simple rules for getting the most out of a summer laboratory internship</i> <br>
        Toby P Aicher and Dániel L Barabási and <b>Benjamin D. Harris</b> and Ajay Nadig and Kaitlin L Williams <br>
        PLOS Computational Biology 13 (8), e1005606, 2017, 2017 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:qjMakFHDy7sC">[Google Scholar]</a>
      </p>

      <p><i>Improving the specificity and efficacy of CRISPR/CAS9 and gRNA through target specific DNA reporter</i> <br>
        Jian-Hua Zhang and Mritunjay Pandey and John F Kahler and Anna Loshakov and <b>Benjamin Harris</b> and Pradeep K Dagur and Yin-Yuan Mo and William F Simonds <br>
        Journal of biotechnology, 2014 <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:d1gkVwhDpl0C">[Google Scholar]</a>
      </p>

      <p><i>Utilizing Stim-RTM Technology to Reduce Irradiated Feeder Cell Use in the Tumor Infiltrating Lymphocyte Culture Process</i> <br>
        Riley Allen and Jason Duecker and Jessica Ojeda and <b>Benjamin Harris</b> and Mikayla Min and Samantha Chan and Christina Ta and Emily Fu-Sum and Purnima Sundar and Yogin Patel and Aileen Li and Shobha Potluri and Suman Vodnala and Gary Lee and Alexander Cheung <br>
        ,  <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=vEHTfVwAAAAJ&citation_for_view=vEHTfVwAAAAJ:_FxGoFyzp5QC">[Google Scholar]</a>
      </p>
      <!-- SCHOLAR_END -->
      </div>
      <h1>Software</h1>
      <p>Python MetaNeighbor : a method to rapidly assess cell type identity using both functional and random gene sets
        <br>
        Github : <a href="https://github.com/gillislab/pyMn">pyMN</a>
      </p>
      <h1>Teaching</h1>
      <h4>Summer 2021 CSHL Intern Python Programming Course</h4>
      <p>
        I contributed 3 lectures to the 10 lecture course, linked below <br>
        <span class="tab"></span><b>Lecture 3: </b><a
          href="https://github.com/bharris12/URP_2021_Programming_Course/tree/main/lecture_3">Pandas</a><br>
        <span class="tab"></span><b>Lecture 5: </b><a
          href="https://github.com/bharris12/URP_2021_Programming_Course/tree/main/lecture_5">Pandas 2 +
          Statistics</a><br>
        <span class="tab"></span><b>Lecture 6: </b><a
          href="https://github.com/bharris12/URP_2021_Programming_Course/tree/main/lecture_6">Genomics</a>
      </p>

      <p>
        I co-organize a seminar series in the CSHL quantiative biology department called "Practical Programming" <br>
        <span class="tab">The seminars have been on topics including <br>
          <span class="tab">Multiprocessing multithreading and vectorization<br>
            Style and Organization in Python<br>
            Elzar Batch System Usage <br>
            Snakemake<br>
            Python packages<br>
            Optimizing for performance/speed (matlab, python)<br>
            Github workflow<br>
            Unit Testing<br>
            Personal Web dev<br></span></span>
        <br>
        <span class="tab">I taught the Multiprocessing multithreading and vectorization lecture. <br>
          <span class="tab"></span>The materials are in this <a
            href="https://github.com/bharris12/practical_progamming_lecture_1">repository</a></span>
      </p>

      <h1>Misc</h1>
      <p>In my free time you can find me in nature...</p>



      <div class="image-gallery">
        <img src="image1.JPG" alt="" class="gallery-img" />
        <img src="image2.JPG" alt="" class="gallery-img" />
        <img src="image3.JPG" alt="" class="gallery-img" />
      </div>
    </section>

    </section>
    </section>
  </div>

</body>

</html>